xyz.slothlife.Jogger.json

```diff
@@ -0,0 +1,100 @@
+{
+    "app-id" : "xyz.slothlife.Jogger",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
```

Comment:
45 isn't out 45beta is available on flathub-beta

Reply:
45 was released today

Reply:
Looks like hasn't been published then:

```
FB: Running 'flatpak --user install -y --noninteractive flathub org.gnome.Platform/x86_64/45' on host
error: Nothing matches org.gnome.Platform in remote flathub
```

I don't see it.

Reply:
Ok _now_ it's there